app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitListFragment.java

```diff
@@ -71,11 +73,13 @@ public View onCreateView(@NonNull LayoutInflater inflater,
             List<DocumentSnapshot> snapshotList = querySnapshotTask.getResult().getDocuments();
             for (int i = 0; i < snapshotList.size(); i++) {
                 Map<String, Object> extractMap = snapshotList.get(i).getData();
+                Log.d(TAG, extractMap.toString());
```

Comment:
Added this as debugging, the id as being queried as habitID and would return null, causing a crash.